docs/modules/migrate/pages/dmt-migration.adoc

```diff
@@ -0,0 +1,49 @@
+= Migrate Your Data
+:description: The DMT migrates your data for maps and replicated maps, compact serialization schemas, and CP data structures. It can be run on Mac, Linux, and Windows Operating Systems.
+
+{description}
+
+NOTE: To migrate Replicated metadata, you must include the provided JAR in your class path. The source cluster must be restarted after the addition of the JAR.
```

Comment:
```suggestion
NOTE: To migrate Replicated metadata and/or CP structures, you must include the provided JAR in your class path. The source cluster must be restarted after the addition of the JAR.
```

Reply:
Also, maybe remove this note as it may be confusing.


or we can give a link to the related place without removing it.